server/msteams/client.go

```diff
@@ -738,8 +738,7 @@ func (tc *ClientImpl) UpdateMessage(teamID, channelID, parentID, msgID, message
 	}
 
 	if updateMessageRequest == nil {
-		tc.logService.Debug("Received nil updateMessageRequest from MS Graph")
-		return nil, errors.New("nil updateMessageRequest")
+		return nil, errors.New("Received nil updateMessageRequest from MS Graph")
```

Comment:
Received nil in updateMessageRequest from MS Graph. Please change this everywhere